Bottom: a5d3e21e9deb196582a9f38f58a3a7a73b00d82b
Top:    4053ebe725505b3967938051182491613ac3c4d4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-11 23:47:42 +0000

Remove unused HistorySize field from the settings structure


---

diff --git a/src/global.h b/src/global.h
index 3794448..19f4454 100644
--- a/src/global.h
+++ b/src/global.h
@@ -668,8 +668,7 @@ typedef struct			/* some resources... */
     /* connections is done */
     AutoPlace;			/* flag which says we should force placement of the
 				   windows on startup */
-  int HistorySize,		/* FIXME? Used in hid/xaw code only. */
-    init_done;
+  int init_done;
 }
 SettingType, *SettingTypePtr;
